### DISCUSSION TITLE
Is there any difference in performance and/or execution when using engineType "binary"?

### DISCUSSION THREAD

**Discussion by zanzlender** [prisma/prisma#18634] [Q&A] [answered]
Hello, I've been having some issues lately with Prisma and Node 18+. I found that a way around it is setting the `engineType = true` flag in the schema. 

While I understand what it is and how it generally works, I would like to know if there is any performance difference and/or difference in the execution when using `binary` over the default `Node-API`?

**Comment by ludralph** [comment, marked as answer]
Hi @zanzlender 👋 

By default, the Prisma Client loads the query engine as a [Node-API library](https://nodejs.org/api/n-api.html). You can alternatively [configure Prisma to use the query engine compiled as an executable binary](https://www.prisma.io/docs/concepts/components/prisma-engines/query-engine#configuring-the-query-engine), which is run as a sidecar process alongside your application. The Node-API library approach is recommended since it reduces the communication overhead between the Prisma Client and the query engine.
You can learn more about this [here](https://www.prisma.io/docs/concepts/components/prisma-engines/query-engine#the-query-engine-at-runtime)

**Reply by ludralph** [reply]
Hi @zanzlender 

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Reply by zanzlender** [reply]
Sry, my notifications were turned off for some reason... But thanks for the answer I understand it better now 👍
